lnrpc/routerrpc/router_backend.go

```diff
@@ -36,6 +45,16 @@ type RouterBackend struct {
 	FindRoute func(source, target route.Vertex,
 		amt lnwire.MilliSatoshi, restrictions *routing.RestrictParams,
 		finalExpiry ...uint16) (*route.Route, error)
+
+	// Activate the debug htlc mode. With the debug HTLC mode, all payments
+	// sent use a pre-determined payment hash.
+	DebugHTLC bool
```

Comment:
is this necessary?

Reply:
I added it with the idea that we eventually are going to remove the main rpc `SendPayment` and that `DebugHTLC` is still useful. Do you think it should be removed?

Reply:
I don't know if anybody uses this anymore. Introducing a new API feels like a good opportunity to get rid of it. Easier to add it later if someone really still want it

Reply:
Removed